profile/README.md

```diff
@@ -14,16 +14,22 @@ The Holoscan platform consists of:
     * [Development container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan)
     * [Debian package](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_dev_deb)
     * [Python wheel](https://pypi.org/project/holoscan)
+    * [Conda package](https://anaconda.org/conda-forge/holoscan)
     * [Source code](https://github.com/nvidia-holoscan/holoscan-sdk)
 * **Holoscan Deployment Stack**: OpenEmbedded/Yocto layer for optimized deployment of applications developed using the Holoscan SDK
     * [OpenEmbedded/Yocto Build container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan-oe-builder)
     * [OpenEmbedded/Yocto Recipes](https://github.com/nvidia-holoscan/meta-tegra-holoscan)
 * **Holoscan Sensor Bridge Software**: integrate the Holoscan Sensor Bridge Device in a Holoscan Application
     * [Source code](https://github.com/nvidia-holoscan/holoscan-sensor-bridge)
     * [Documentation](https://docs.nvidia.com/holoscan/sensor-bridge/latest)
+* **Holoscan Networking**: packaging of Holoscan [Basic](https://github.com/nvidia-holoscan/holohub/tree/main/operators/basic_network) and [Advanced Networking Operators](https://github.com/nvidia-holoscan/holohub/tree/main/operators/advanced_network)
```

Comment:
We're not listing ano other holohub operators in this page. Any reason why we'd do this just for these?

Reply:
There actually is a link to holohub at the top: https://github.com/nvidia-holoscan/.github/pull/6/files#diff-0e83982cf6f4dabedcbf7b12029a56c6f4a728064ba99543fff8f227c0654b45R9